package.json

```diff
@@ -22,12 +22,12 @@
   "dependencies": {
     "eslint-plugin-node": "11.1.0",
     "eslint-plugin-security": "1.4.0",
-    "eslint-plugin-react": "7.33.0"
+    "eslint-plugin-react": "7.36.1"
```

Comment:
Is there a reason for why we can't use a wider version range here? It's causing duplicates to be installed in our repos.

Reply:
We will also need to provide [flat configs](https://eslint.org/blog/2022/08/new-config-system-part-2/) to be fully compatible with ESLint 9.x